common/xbps-src/shutils/common.sh

```diff
@@ -350,7 +350,13 @@ setup_pkg() {
     fi
 
     # Check if required vars weren't set.
-    _vars="pkgname version short_desc revision homepage license"
+    _vars="pkgname version short_desc revision homepage"
+
+    # Only require license to be set when building metapackages
+    if [ "$build_style" != "meta" ]; then
```

Comment:
Can we also do a check to confirm no files are provided?

Reply:
That will problably go on common/hooks/pre-pkg/99-pkglint,sh

but there already is a check for empty DESTDIR while not being build_style=meta https://github.com/void-linux/void-packages/blob/master/common/hooks/pre-pkg/99-pkglint.sh#L36